src/models/BaseModel.py

```diff
@@ -38,6 +39,11 @@ def __init__(self, train_size=0.75, test_size=None):
             print('Pre-processed data exists, reading from the file')
             self.cleaned_encoded_data = self.load_cleaned_data()
             print('Data read complete')
+            if force_load_all:
```

Comment:
the problem is, that the `cleaned_encoded_data` does not contain the raw description data we need for the similarity model here. 

all other models are fine with the cleaned_encoded_data, so using the force_load_all you can force it to load (at least) the raw description data